### PR TITLE
made changes to display the phone number under SignIn Information

### PR DESCRIPTION
**Pull request by ramavavi**
Git hub Issue: https://github.com/episphere/connectApp/issues/349

Under the Sign In Information, it now displays the phone number or email used while logging in.

### CODE REVIEW

File: js/pages/settings.js [hunk 1]
```diff
@@ -296,4 +252,79 @@ const changeEmail = async (newEmail) =>{
 
     });
 
+}
+
+
+const displayFirebaseInfo = (userData) => {
+    let signInInfornmationContent = ``
+    // write conditions for email, phone or empty
```
Comment (Gbarra9) [inline js/pages/settings.js:260]: This code comment in my opinion is not needed.

File: js/pages/settings.js [hunk 2]
```diff
@@ -296,4 +252,79 @@ const changeEmail = async (newEmail) =>{
...
+    
+    //// if firebase authentication email number exists
+    console.log("userData", userData)
```
Comment (Gbarra9) [inline js/pages/settings.js:263]: This console log can be removed.

**Review by Gbarra9 — CHANGES_REQUESTED** [review]

**Comment by anthonypetersen** [comment]
please modify the PR, this can't go directly into production

File: js/pages/settings.js [hunk 3]
```diff
@@ -296,4 +252,79 @@ const changeEmail = async (newEmail) =>{
...
+    if(userData['421823980']) {
+        signInInfornmationContent =
```
Comment (anthonypetersen) [inline js/pages/settings.js:265]: correct spelling

**Review by anthonypetersen — CHANGES_REQUESTED** [review]
correct spelling of variable, change target branch of PR

**Review by anthonypetersen — APPROVED** [review]

**Pull request merged**